frontend/src/scenes/sessions/filters/SessionsEditFiltersPanel.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { Button, Card, Col, Row } from 'antd'
+import { useActions, useValues } from 'kea'
+import { DownOutlined, PlusCircleOutlined } from '@ant-design/icons'
+import { CloseButton } from 'lib/components/CloseButton'
+import { EventTypePropertyFilter, PersonPropertyFilter, RecordingPropertyFilter } from '~/types'
+import { sessionsFiltersLogic } from 'scenes/sessions/filters/sessionsFiltersLogic'
+import { EventPropertyFilter } from 'scenes/sessions/filters/EventPropertyFilter'
+import { PersonFilter } from 'scenes/sessions/filters/UserFilter'
+import { DurationFilter } from 'scenes/sessions/filters/DurationFilter'
+
+interface Props {
+    onSubmit: () => void
+}
+
+const SECTIONS: Record<string, { label: string; description: string }> = {
+    recording: {
+        label: 'Recording filters',
+        description: 'Find sessions with recordings matching the following',
+    },
+    person: {
+        label: 'User property filters',
+        description: 'Find sessions where user properties match the following',
+    },
+    action_type: {
+        label: 'Action filters',
+        description: 'Find sessions where user has done a given action',
+    },
+    event_type: {
+        label: 'Event filters',
+        description: 'Find sessions where user has done a given event',
+    },
+    cohort: {
+        label: 'Cohort filters',
+        description: 'Find sessions by users in the following cohorts',
+    },
+}
+
+export function SessionsEditFiltersPanel({ onSubmit }: Props): JSX.Element {
```

Comment:
Nit?: I found the name for this component a bit confusing.

Reply:
Solved